provider.tf

```diff
@@ -0,0 +1,5 @@
+provider "aws" {
+  default_tags {
+    tags = var.tags
```

Comment:
I think we need to do something like this:
default_tags {
    Project     = var.project
    Environment = var.environment
  }